nntrainer/layers/fc_layer.cpp

```diff
@@ -115,6 +150,18 @@ void FullyConnectedLayer::setProperty(const std::vector<std::string> &values) {
   LayerImpl::setProperty(remain_props);
 }
 
+/**
+ * @brief forwarding function for lora. 
+ * It would be called during `forwarding` of FC layer.
+*/
+void FullyConnectedLayer::forwarding_lora(RunLayerContext &context, Tensor &weight){
```

Comment:
If we are going to support LoRA this function will be needed all of layers not just fc layer so please consider to make it in parent class.

Reply:
Exactly! In this PR, I wrote the code only for fc layer, but it is required to make a parent class to cover LoRA in general way. I'm gonna reflect this comment. Thanks.